src/solvers/flattening/bv_pointers.cpp

```diff
@@ -457,19 +457,28 @@ bvt bv_pointerst::convert_pointer_type(const exprt &expr)
   return conversion_failed(expr);
 }
 
+static bool is_pointer_subtraction(const exprt &expr)
+{
+  if(expr.id() != ID_minus)
+    return false;
+
+  const auto &minus_expr = to_minus_expr(expr);
+
+  return minus_expr.op0().type().id() == ID_pointer &&
```

Comment:
`lhs()` and `rhs()`?

Reply:
`minus_exprt` doesn't actually have these -- I'll do a separate PR to add.